README.md

```diff
@@ -19,18 +19,10 @@ The device object internally uses an instance of the D3D12TranslationLayer immed
 
 ## Building
 
-This project is expected to be included in a CMake build environment where the D3D12TranslationLayer project is also included. Additionally, the [WDK](https://docs.microsoft.com/en-us/windows-hardware/drivers/download-the-wdk) (Windows Driver Kit) must be installed, in order to provide `d3d10umddi.h` to D3D11On12, and in order to generate the D3D12TranslationLayer_WDK project, which hosts some code required to parse DXBC shaders and containers.
+In order to build D3D11On12, the [WDK](https://docs.microsoft.com/en-us/windows-hardware/drivers/download-the-wdk) (Windows Driver Kit) must be installed, in order to provide `d3d10umddi.h` to D3D11On12, and in order to generate the D3D12TranslationLayer_WDK project, which hosts some code required to parse DXBC shaders and containers. The D3D12TranslationLayer and it's subprojects, D3D12TranslationLayer_WDK and DXBCParser, will be fetched from GitHub when building with CMake if D3D12TranslationLayer_WDK isn't already included. 

```

Comment:
Nitpick: "it's" => "its" in the second-last sentence.

I'd suggest adding some kind of mention of how `D3D12TranslationLayer_WDK` and `DXBCParser` could be already-included - maybe something like ", such as by a parent CMakeLists.txt that has already entered that project"